apps/docs/src/app/structure.ts

```diff
@@ -0,0 +1,999 @@
+import { DateTime } from 'luxon';
+import { DocsLocale } from './constants/locale';
+
+export type DocsDocStructureCategoryItem = {
```

Comment:
В этом и в подобных местах кажется, что DocsDoc - избыточно, одного префикса достаточно
```suggestion
export type DocsStructureCategoryItem = {
```